primo/data_parser/well_data.py

```diff
@@ -933,12 +940,10 @@ def _process_data(self, metrics: SetOfMetrics):
                 )
                 raise_exception(msg, ValueError)
 
-    def _process_dac_data(self):
-        """
-        processes the dac data
-        """
-        self._append_fed_dac_data()
-        return
+    def _compute_fed_dac_score(self):
+        """Appends federal DAC data and computes its score"""
+        # Append Tract ID/GEOID, population density, Total population, land area,
+        # federal DAC score to the DataFrame
```

Comment:
This is missing append_fed_dac_data method. I suspect the branch was forked from an older version of main and there may be other missing changes.